telematic_system/scripts/log_analysis/parse_messaging_server_logs.py

```diff
@@ -69,7 +69,7 @@ def parseInfluxfile(logname, start_time_epoch, end_time_epoch, run_num):
                # Get write time 
                 write_time_split = log_line.split("INFO")
                 write_time_string = write_time_split[0][:len(write_time_split[0]) - 2]
-                log_time_in_datetime = datetime.datetime.strptime(write_time_string, '%Y-%m-%d %H:%M:%S.%f')
+                log_time_in_datetime = datetime.datetime.strptime(write_time_string, '%Y-%m-%dT%H:%M:%S.%fZ')
```

Comment:
What is the Z at the end supposed to be? strptime documentation https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior defines %Z as Time zone name which seems like it shouldn't be required